OOVPA.h

```diff
@@ -98,10 +104,10 @@ typedef struct _LOVP
 #define XREF_ENTRY( Offset, XRef)    \
     { Offset, XRef }
 
-// UNUSED Macro for storing a normal (non-XRef) {Offset, Value}-Pair
+// Macro for storing a normal (non-XRef) {Offset, Value}-Pair
 // Offsets can go up to 16 bits, values are always one byte (8 bits)
 #define OV_ENTRY(Offset, Value)    \
-    { Offset, Value }
+    { Offset, { .unused=0, .cValue=Value } }
```

Comment:
One could use the .unused into a flags field, so it would be possible to do:

```
MATCH(0x0000, 0x01, 0xXX, 0x02, 0x03)
```

which has a wildcard, which is encoded as field that's being ignored (the same could obviously be used for X-Ref references by storing flags bits only in the highest bits; one could still have 14 bits for the X-Ref index, with 2 bits as type [ignored/x-ref/value/<reserved>]).

Anyhow, this needs some more consideration probably

Reply:
(won't fix as nothing is implemented in this pull request)